CHANGELOG.md

```diff
@@ -29,6 +29,10 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ### Fixed
 - Inform about over-cropping action metric timelines. Fix [JPERF-152].
```

Comment:
I think we also need to describe what was fixed here :)
>- Stop ignoring OutlierTrimming in MaximumCoverageJudge. Fix [JPERF-443].

Reply:
Done